### ISSUE TITLE
[REQ] Generator: Documentation -> LLM / RAG compatible output

### ISSUE THREAD

**Issue by astanciu** [OpenAPITools/openapi-generator#20382]
Would be great to have a generator (probably under the Documentation section) that outputs a format friendly to LLMs to help with RAG related tasks. The current `markdown` one is pretty close, but has a lot of noise

**Comment by wing328** [comment]
> that outputs a format friendly to LLMs to help with RAG related tasks. 

agreed. I talked to many developers and they're already using LLMs to enhance the auto-generated code.

Is there any particular format that's LLM/RAG friendly?

> The current markdown one is pretty close, but has a lot of noise
